Development.md

```diff
@@ -0,0 +1,250 @@
+# Convox Rack Development Guide
+
+Rack is open source with the goal of enabling everyone to understand the platform and to sugguest and contribute improvements.
+
+This guide documents how to:
+
+* Set up a sandbox development Rack to develop and test infrastructure template changes
+* Set up a local development VM to run, develop and test API changes
+* Run the Rack unit test suite locally
+* Submitting changes upstream
+* Run an AWS integration test suite locally or on a CI server
+* Releasing artifacts to enable `convox rack update`
+
+## Sponsorship and Mentoring
+
+Developing Rack will incur AWS costs. If this is an obstacle for you to contribute you can contact support@convox.com to request sponsorship.
```

Comment:
Can you do `mailto` links for emails in markdown? `[support@convox.com](mailto:support@convox.com)`?

Not sure if that works.